charts/nri-bundle/Chart.yaml

```diff
@@ -45,13 +45,21 @@ dependencies:
     version: 1.0.0
 
   - name: kube-state-metrics
-    version: 4.23.0
+    version: 4.25.0
     condition: ksm.enabled,kube-state-metrics.enabled
     repository: https://prometheus-community.github.io/helm-charts
+    alias: kube-state-metrics
+
+  - name: kube-state-metrics
+    version: 4.23.0
+    condition: ksm_v26.enabled,kube-state-metrics-v26.enabled
```

Comment:
The original `kube-state-metrics` dependnecy has two conditions for backwards compatibility reasons. `ksm.enabled` is deprecated, so I wouldn't propagate that pattern to new dependencies. I'd suggest to use the chart name (or alias) plus `.enabled` as the only condition.